src/webgpu/api/validation/encoding/queries/occlusion.spec.ts

```diff
@@ -1,14 +0,0 @@
-export const description = `
```

Comment:
It's a bit weird to have `general.spec.ts` and `begin_end.spec.ts` when the tests are basically different for all query types. But that matches the test plan so let's do that for now and see if we need to resplit tests later.